pandas/util/_test_decorators.py

```diff
@@ -37,7 +37,7 @@ def test_foo():
 from pandas.core.computation.expressions import _NUMEXPR_INSTALLED, _USE_NUMEXPR
 
 
-def safe_import(mod_name, min_version=None):
+def safe_import(mod_name: str, min_version: Optional[str] = None):
```

Comment:
```suggestion
def safe_import(mod_name: str, min_version: Optional[str] = None) -> Union[types.ModuleType, bool]:
```

Actually not 100% sure on this one but would be nice if return type could be annotated as well

Reply:
Code style question, what is preferred here?

```from types import ModuleType````

or

```import types```

Reply:
NVM, mypy complains about it anyway.